components/grid.css

```diff
@@ -0,0 +1,31 @@
+.grid-two,
+.grid-three {
+    display: grid;
+    grid-gap: 2rem;
+    justify-content: center; 
+     align-items: center;
```

Comment:
Use prettier it helps to maintain the code, else everything looks good.

Reply:
Thanks for reviewing , I am already using prettier for my code,  that line 6 I added at end that's why it's not in same alignment